cmd/syncthing/main.go

```diff
@@ -264,6 +264,24 @@ func parseCommandLineOptions() RuntimeOptions {
 	return options
 }
 
+type exiter struct{}
```

Comment:
Control or Controller or App? Also, interface that lives somewhere common oppose to oppurtunistic interface matching?